### ISSUE TITLE
Accessibility bug: DetailsList column header sorted or filter icon does not support screen reader

### ISSUE THREAD

**Issue by inateeg** [OfficeDev/office-ui-fabric-react#6761]
### Please provide a reproduction of the bug in a codepen:
 go to:  aka.ms/itemwview  ==> SP Item View page you can see the column.  then click on name column header to select A to Z sorting to see sort icon appears in the header but screen reader does not announce it.

DetailsList column header where there is sorted or filter applied icon that does not support screen reader because it has <I … aria-hidden="true">.  User would like to know if the sort for filter applied to the column through aria-lable of those icons.
 
 
- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: (fill this out if relevant)
 
Currently SPO accessibility support take high priority.
 
Are you willing to submit a PR to fix? (Yes, No)
 Proposed change could be if isSorted or isFiltered true, when render those <Icon> always add ariaLabel to <Icon> props so that anything visually can be seen, has ariaLabel.  If fabric team agree with this change, in file DetailsColumn.base.tsx around line 126 to 132... I don't mind to provide PR.  Otherwise fabric team can decide the better way?
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: (if applicable)
any SPO Documents library page

or aka.ms/itemsview page then go to SP Items View page
 
#### Describe the issue:
 go to:  aka.ms/itemwview  ==> SP Item View page you can see the column.  then click on name column header to select A to Z sorting to see sort icon appears in the header but screen reader does not announce it.
 
#### Actual behavior:
 screen reader only read column header name
 
#### Expected behavior:
 
Screen reader can read column header name and if there are any icons indicating sort or filter attributes

**Comment by inateeg** [comment]
after play more with not so smart narrator I found that even though icon element does not have aria-label but outter <div> that marked as columnheader has aria-sort=true set and if you let narrator in scan mode it would read the column is sort ascending.  that should be enough.
